components/wrong-network-modal.tsx

```diff
@@ -25,30 +26,69 @@ export default function WrongNetworkModal({requiredNetwork = ``}) {
     return `danger`
   }
 
+
   function getColumnClass() {
     const color = getColor();
 
     return `col-6 rounded border border-${color} text-${color} p-3 d-flex justify-content-between align-items-center`;
   }
 
-  return <Modal title="Change network" show={showModal()}>
-    <div className="text-center">
-      <strong className="capition d-block text-uppercase text-white-50 mb-4">
-        please, connect to the <span className="text-purple">kovan test network</span> on your metamask wallet
-      </strong>
-      <div className="row">
-        <div className="col-12 d-flex justify-content-center">
-          <div className={getColumnClass()}>
-            <div className="d-flex justify-content-start align-items-center">
-              <Image src={metamaskLogo} width={15} height={15}/> <span className="ms-2">{truncateAddress(currentAddress)}</span>
-            </div>
-            {!showModal() ? <CheckMarkIcon /> : <ErrorMarkIcon/>}
+  return (
+    <Modal
+      title="Connect your MetaMask Wallet"
```

Comment:
Wrong Modal component **CAN'T** return a "connect your metamask" component.

Reply:
But, it's the new design: https://www.figma.com/file/yQITClZ8qyxHBDwPk4gHgC/BEPRO-Network?node-id=2241%3A5611

Reply:
https://www.figma.com/file/yQITClZ8qyxHBDwPk4gHgC/BEPRO-Network?node-id=2241%3A5611 this is the modal you should be showing